it/connectors/DeleteSavingsIncomeDataTysConnectorISpec.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import com.github.tomakehurst.wiremock.http.HttpHeader
+import config.{AppConfig, BackendAppConfig}
+import helpers.WiremockSpec
+import models.{ErrorBodyModel, ErrorModel}
+import org.scalatestplus.play.PlaySpec
+import play.api.Configuration
+import play.api.http.Status._
+import uk.gov.hmrc.http.{HeaderCarrier, HeaderNames, HttpClient, SessionId}
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import utils.TaxYearUtils.{convertSpecificTaxYear, specificTaxYear}
+
+class DeleteSavingsIncomeDataTysConnectorISpec extends PlaySpec with WiremockSpec{
+
+  lazy val connector: DeleteSavingsIncomeDataTysConnector = app.injector.instanceOf[DeleteSavingsIncomeDataTysConnector]
+  implicit val hc: HeaderCarrier = HeaderCarrier()
+  val nino = "nino"
+
+  val desUrl = s"/income-tax/income/savings/${convertSpecificTaxYear(specificTaxYear)}/$nino"
+
+  lazy val httpClient: HttpClient = app.injector.instanceOf[HttpClient]
+
+  def appConfig(desHost: String): AppConfig = new BackendAppConfig(app.injector.instanceOf[Configuration], app.injector.instanceOf[ServicesConfig]) {
+    override val desBaseUrl: String = s"http://$desHost:$wireMockPort"
+  }
+
+  "DeleteSavingsIncomeDataTysConnector " should {
+
+    "include internal headers" when {
+      val headersSentToDes = Seq(
+        new HttpHeader(HeaderNames.authorisation, "Bearer secret"),
+        new HttpHeader(HeaderNames.xSessionId, "sessionIdValue")
+      )
+
+      val internalHost = "localhost"
+      val externalHost = "127.0.0.1"
+      implicit val hc: HeaderCarrier = HeaderCarrier(sessionId = Some(SessionId("sessionIdValue")))
+
+      "the host for DES is 'Internal'" in {
```

Comment:
We should probably change references of DES to IF as this TYS API is an IF API.  I think it just makes it a little less confusing when reading it later.

Reply:
Yep agreed, I'll make the change in the next commit